models/Student.js

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose'
+const {Schema, model} = mongoose
+
+const StudentSchema = new Schema({
+    firstname: {type: String, required: true},
+    lastname: {type: String, required: true},
+    favQuote: {type: String, required: false},
+    role: { type: String,
+            enum: 
+            {
+                values: ['User', 'Admin'],
+                message: '{VALUE} is not supported'
```

Comment:
Nice to add a custum validation message! 
Indeed, the default mongoose validation errors suck. So very appreciated :+1: